cdap-elastic/src/main/java/io/cdap/cdap/metadata/elastic/ElasticsearchMetadataStorage.java

```diff
@@ -184,31 +186,21 @@
   @Inject
   public ElasticsearchMetadataStorage(CConfiguration cConf) {
```

Comment:
Can we also add NonNull checker for cConf.

Reply:
We assume arguments are not null, unless it is annotated with `@Nullable`. Also, guice won't inject null in constructor.